docs/solutions/postgis-deploy.md

```diff
@@ -0,0 +1,105 @@
+# Deploy spatial data with PostgreSQL
+
+The following document provides guidelines how to install PostGIS and how to run the basic queries. 
+
+## Preconditions
+
+1. We assume that you have the basic knowledge of spatial data, GIS (Geographical Information System) and of shapefiles.
+2. You need to acquire spacial data. For the following examples, we'll use the same [data set](https://s3.amazonaws.com/s3.cleverelephant.ca/postgis-workshop-2020.zip) as is used in [PostGIS tutorial](http://postgis.net/workshops/postgis-intro/) 
+
+## Install PostGIS
+
+1. Enable Percona repository.
+
+    As other components of Percona Distribution for PostgreSQL, PostGIS is available from Percona repositories. Use the [`percona-release`](https://docs.percona.com/percona-software-repositories/installing.html) repository management tool to enable the repository. 
+
+    ```{.bash data-prompt="$"}
+    $ sudo percona-release setup ppg15
+    ```
+
+2. Install PostGIS packages
+
+    === "On Debian and Ubuntu"
+
+          ```{.bash data-prompt="$"}
+          $ sudo apt install percona-postgis
+          ```
+
+    === "On RHEL and derivatives"
+
+          ```{.bash data-prompt="$"}
+          $ sudo yum install percona-postgis33
+          ```
+
```

Comment:
Also it is needed to install epel repository
 sudo yum install epel-release
 Also on ol9 and ol8 it is needed to enable `code ready builder` repository
 sudo dnf config-manager --set-enabled ol8_codeready_builder - ol8
 sudo dnf config-manager --set-enabled ol9_codeready_builder - ol9